config/config_test.go

```diff
@@ -278,6 +278,7 @@ spilled-file-encryption-method = "plaintext"
 	c.Assert(conf.Security.SpilledFileEncryptionMethod, Equals, SpilledFileEncryptionMethodPlaintext)
 	c.Assert(conf.DeprecateIntegerDisplayWidth, Equals, true)
 	c.Assert(conf.EnableEnumLengthLimit, Equals, false)
+	c.Assert(conf.EnableForwarding, Equals, false)
```

Comment:
In this test, you should try to set the value to a non-default value in the config text above, and assert it's actually set to a non-default value. So you should add `enable-forwarding=true` at about L200 and assert it's true here.